test/subset/run-tests.py

```diff
@@ -51,10 +57,7 @@ def run_test (test, should_check_ots):
 		    "--drop-tables-=sbix"]
 	cli_args.extend (test.get_profile_flags ())
 	print (' '.join (cli_args))
-	_, return_code = cmd (cli_args)
-
-	if return_code:
-		return fail_test (test, cli_args, "%s returned %d" % (' '.join (cli_args), return_code))
```

Comment:
You are not tracking individual failures anymore. That's not very friendly. Let's read back the status.

Reply:
If hb-subset failed, it will be caught when comparing the output with the expected on. The shaping tests don't check the return value either.

Reply:
Well. Shape results are communicated via stdout. Subset results are communicated via a file. What if the file exists from previous runs, but subset fails now without removing the old file?

Reply:
The test runner is using temporary folders, so this shouldn't happen. But lets be more through and communicate the failure right away.

Reply:
Thanks.

Reply:
Done.